keeper/src/main/scala/zio/keeper/errors.scala

```diff
@@ -15,16 +13,28 @@ sealed abstract class SerializationError(msg: String = "") extends Error(msg = m
 
 object SerializationError {
 
-  final case class SerializationTypeError[A](val cause: Throwable)(implicit ct: ClassTag[A])
+  final case class SerializationTypeError(msg0: String)
```

Comment:
we should probably also set the exception cause in order to preserve stack traces

Reply:
I will add that